packages/types/src/options.ts

```diff
@@ -66,7 +66,10 @@ export interface Options {
   /** Sets the distribution for all events */
   dist?: string;
 
-  /** The maximum number of breadcrumbs sent with events. Defaults to 100. */
+  /** 
+  * The maximum number of breadcrumbs sent with events. Defaults to 30.
```

Comment:
Trying out the "suggest a change" workflow here.

```suggestion
   * The maximum number of breadcrumbs sent with events. Defaults to 30.
```